ims/exception/config_exceptions.py

```diff
@@ -25,3 +25,17 @@ def __init__(self, section):
 
     def __str__(self):
         return "Missing " + self.option + " section in config file"
+
+
+class InvalidValueConfigException(ConfigException):
```

Comment:
@chemistry-sourabh add the docstring on this one, then this should be ready to merge.